.circleci/config.yml

```diff
@@ -157,22 +151,10 @@ jobs:
       - run:
           name: Integration tests (Bigtable)
           command: make integration-test
-          environment:
-            DB_DSN: grpc://localhost:8086
-            TEST_RESULTS_DIR: workspace/test-results
-      - run:
-          name: Integration tests (DynamoDB)
-          command: make integration-test
-          environment:
-            TEST_RESULTS_DIR: workspace/test-results
-      - run:
-          name: Integration tests (Dual Bigtable/DynamoDB)
-          command: make integration-test
           environment:
             RUST_LOG: autopush=debug,autopush_common=debug,autoendpoint=debug,autoconnect=debug,slog_mozlog_json=info,warn
             # PYTEST_ARGS: -sv
-            DB_DSN: dual
-            DB_SETTINGS: tests/integration/dual_test.json
+            DB_DSN: grpc://localhost:8086
```

Comment:
Note for me: 
the `test_integration_all_rust.py` already sets the table names via an env var.

Reply:
Yeah good point. Perhaps considering the removal of that in the tests so we're explicit about config might be wise, however I think whatever we pass in via an env var here overrides whatever may be set in the test? Or not?